index.js

```diff
@@ -262,6 +299,12 @@ class GhostInspector {
       callback = options
       options = {}
     }
+    options = options || {}
+    // we can poll if there's no CSV and immediate=0
+    const canPoll = !options.immediate && !options.dataFile
```

Comment:
Is `immediate` always defined? Just checking if `null` or `undefined` would be a problem here

Reply:
`options.immediate // undefined` is probably the default. The default behaviour of the API is to maintain an open HTTP request and wait for execution to complete. To get an immediate response you are required to pass `immediate=1`, so any truthy value here will mean we don't want to poll.

Does that clarify things?

Reply:
Yes, that does, thanks!